app/controllers/spree/orders_controller.rb

```diff
@@ -10,6 +10,7 @@ class OrdersController < ::BaseController
     before_action :check_authorization
     rescue_from ActiveRecord::RecordNotFound, with: :render_404
     helper 'spree/products', 'spree/orders'
+    helper 'terms_and_conditions'
```

Comment:
I don't think that this is needed. This is inheriting from the BaseController which inherits from ApplicationController. There were a couple of view specs failing here but I fixed them by adding the helper to the spec because view specs don't load the controller.

Reply:
Ah yes, sorry - I added that in an attempt to fix those last two specs and didn't check back on the CI to see if it worked. (The specs were obviously passing locally). We can revert that commit.

Reply:
Okay, I removed that commit. All tests are passing. I'm moving this to test ready.